LayoutKit/Views/StackView.swift

```diff
@@ -83,6 +83,27 @@ open class StackView: UIView {
         setNeedsLayout()
     }
 
+    /**
+     Deletes all subviews from the stack.
+     */
+    open func removeArrangedSubviews() {
+        arrangedSubviews.removeAll()
+        invalidateIntrinsicContentSize()
+        setNeedsLayout()
+    }
+
+    /**
+     Deletes all subviews from the stack.
+     */
+    open func removeArrangedSubviews() {
+        for subview in subviews {
```

Comment:
given the name, this should really only iterate over the views in arrangedSubviews, not all subviews.